share/cmake/modules/FindExtPackages.cmake

```diff
@@ -195,14 +190,31 @@ if((OCIO_BUILD_APPS AND OCIO_USE_OIIO_FOR_APPS) OR OCIO_BUILD_TESTS)
     # OpenImageIO
     # https://github.com/OpenImageIO/oiio
     set(OIIO_VERSION "2.2.14")
-
-    if(OCIO_USE_OIIO_CMAKE_CONFIG)
-        # TODO: Try when OIIO 2.4 is released (https://github.com/OpenImageIO/oiio/pull/3322).
-        # set(OPENIMAGEIO_CONFIG_DO_NOT_FIND_IMATH 1)
-        find_package(OpenImageIO ${OIIO_VERSION} CONFIG)
-    else()
-        find_package(OpenImageIO ${OIIO_VERSION})
-    endif()
+    set(OIIO_RECOMMENDED_VERSION "2.4")
+
+    # Supported from OIIO 2.4+. Setting this for lower versions doesn't affect anything.
+    set(OPENIMAGEIO_CONFIG_DO_NOT_FIND_IMATH 1)
+
+    ###############################################################################
+    # OpenEXR (https://github.com/AcademySoftwareFoundation/openexr)
```

Comment:
OpenImageIO?

Reply:
Yes, sorry, copy/paste error.  We will fix.  Thanks for the review!